404-handler/main_test.go

```
@@ -0,0 +1,57 @@
+	app := setupApp()
+
+	// Create a test request
+	req := httptest.NewRequest(http.MethodGet, "/hello", nil)
+	resp, _ := app.Test(req, -1) // -1 disables timeout
+
+	// Check the response
+	assert.Equal(t, http.StatusOK, resp.StatusCode)
+
+	// Read the response body
+	body := make([]byte, resp.ContentLength)
+	resp.Body.Read(body)
```

golangci-lint (1.62.2)

Error return value of `resp.Body.Read` is not checked

(errcheck)